src/types/json.h

```diff
@@ -217,6 +217,25 @@ struct JsonValue {
     return results;
   }
 
+  StatusOr<Optionals<uint64_t>> StrBytes(std::string_view path) const {
+    Optionals<uint64_t> results;
+    try {
+      jsoncons::jsonpath::json_query(
+          value, path, [&results](const std::string & /*path*/, const jsoncons::json &origin) {
+            if (!origin.empty()) {
+              // This is only a rough calculation of the size of the string size char, not the entire byte occupied by
+              // the object
+              results.emplace_back(origin.as_string().size() * sizeof(std::string::value_type));
```

Comment:
Could this json_query matches more than 1 element?

Reply:
If multiple elements are hit, an array is returned. This is the return value of Redis 7.0 
<img width="453" alt="251ef557533affef376c009aaf4d218" src="https://github.com/apache/kvrocks/assets/127465317/0461b0fc-fc59-49ec-b418-bfd2029d6826">